pkg/infra/iac2/templates/aws_helm_chart/factory.ts

```diff
@@ -0,0 +1,47 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as pulumi_k8s from '@pulumi/kubernetes'
+
+interface Args {
+    Name: string
+    Directory?: string
+    Chart?: string
+    FetchOpts?: pulumi.Input<pulumi_k8s.helm.v3.FetchOpts>
+    Values?: pulumi.Inputs
+    Version?: string
+    Namespace?: string
+    KubernetesProvider: pulumi_k8s.Provider
+    dependsOn: pulumi.Input<pulumi.Input<pulumi.Resource>[]> | pulumi.Input<pulumi.Resource>
+}
+
+// noinspection JSUnusedLocalSymbols
+function create(args: Args): pulumi_k8s.helm.v3.Chart {
```

Comment:
the name of aws_helm_chart is a bit weird just because we can deploy non aws helm charts to an eks cluster, but definitely not a blocker. Im just wondering if we can have something more generic, but because eks is our only kubernetes provider to date im ok with this and try to learn from it